variation/schemas/gnomad_vcf_to_protein_schema.py

```diff
@@ -0,0 +1,14 @@
+"""Module for gnomad vcf to protein response schema"""
+from typing import Optional
+
+from ga4gh.core import core_models
+from pydantic import StrictStr
+
+from variation.schemas.normalize_response_schema import NormalizeService
+
+
+class GnomadVcfToProteinService(NormalizeService):
+    """Define response for gnomad vcf to protein service"""
+
+    gene_context: Optional[core_models.Gene] = None
+    vrs_ref_allele_seq: Optional[StrictStr] = None
```

Comment:
I could have just added this to variation extensions field, but thought it'd be easier for varcat if we had a field to retrieve

Reply:
I am always pro making varcat better (easier to develop). @ahwagner any objections?